cms/djangoapps/contentstore/signals/handlers.py

```diff
@@ -159,11 +160,46 @@ def listen_for_course_publish(sender, course_key, **kwargs):  # pylint: disable=
     transaction.on_commit(lambda: emit_catalog_info_changed_signal(course_key))
 
 
+def _determine_producer_config_for_signal_and_topic(signal, topic):
+    """
+    Utility method to determine the setting for the given signal and topic in EVENT_BUS_PRODUCER_CONFIG
+
+    Records to New Relic for later analysis.
+
+    Parameters
+        signal (OpenEdxPublicSignal): The signal being sent to the event bus
+        topic (string): The topic to which the signal is being sent
```

Comment:
Can you document which topic names include or exclude the prefix, and ensure that everything is consistent?

Reply:
[fixed]